spring-cloud-data-rest-resource/src/main/java/org/springframework/cloud/data/rest/resource/TaskDefinitionResource.java

```diff
@@ -30,6 +31,12 @@
 
 	private String dslText;
 
+	/**
+	 * Stream status (i.e. running, complete, etc).
+	 */
+	private String status = "";
```

Comment:
for `StreamDefinitionResource` the status is null by default instead of empty string (should both default to "unknown" instead? cc @pperalta)